catalog/app/containers/Bucket/ListingActions.tsx

```diff
@@ -210,7 +211,7 @@ function useMatchedParams(to: string) {
         }
       }
       default:
-        return {}
+        assertNever(match.path as never)
```

Comment:
this statement doesn't do anything bc you virtually just telling it to assert that never explicitly casted by you is never, and this will always succeed no matter what. so you either have to remove the type cast (`as ...`) or replace it with a runtime check (assert / invariant)